modules/aruco/include/opencv2/aruco/charuco.hpp

```diff
@@ -285,7 +287,8 @@ CV_EXPORTS_W void detectCharucoDiamond(InputArray image, InputArrayOfArrays mark
                                        InputArray markerIds, float squareMarkerLengthRate,
                                        OutputArrayOfArrays diamondCorners, OutputArray diamondIds,
                                        InputArray cameraMatrix = noArray(),
-                                       InputArray distCoeffs = noArray());
+                                       InputArray distCoeffs = noArray(),
+                                       Ptr<Dictionary> dictionary = getPredefinedDictionary(PREDEFINED_DICTIONARY_NAME(0)));
```

Comment:
> `Ptr<Dictionary>`

@AleksandrPanov Why is not a const reference?

Reply:
`const Dictionary&` does not work for python binding.
Or do you suggest `const Ptr<Dictionary>&`?